.github/workflows/generate-package-versions.yml

```diff
@@ -99,4 +99,30 @@ jobs:
           base: main
           title: "chore: update ${{ env.VENV_NAME }} latest version to ${{ env.NEW_LATEST }}"
           labels: changelog/no-changelog
-          body-path: .github/PULL_REQUEST_TEMPLATE.md
+          body: |
+            Update ${{ env.VENV_NAME }} lockfiles and dependency package lockfiles.
+            This performs the following updates:
+              1) Some ${{ env.VENV_NAME }} lockfiles use ${{ env.VENV_NAME }} `latest`. This will update ${{ env.VENV_NAME }} and dependencies.
```

Comment:
Riot venv names aren't guaranteed to map to PyPI package names. Not sure if you've taken that into account here.

Reply:
The environment variable name is incorrect and it should more appropriately be named package_name. Thanks for the callout